yolox/utils/logger.py

```diff
@@ -98,8 +98,11 @@ def setup_logger(save_dir, distributed_rank=0, filename="log.txt", mode="a"):
 
     logger.remove()
     save_file = os.path.join(save_dir, filename)
+    crnt_log_save_file = os.path.join(save_dir, 'train_log_crnt.txt')
```

Comment:
Why this `train_log_crnt.txt` is needed?  Seems that your code redirect io to it and remove this file if it exists.

Reply:
@FateScript As you have suggested in your earlier review that you recommend creating a new log file to be uploaded. it is deleted at that start to keep only current run logs in this.
![image](https://github.com/Megvii-BaseDetection/YOLOX/assets/16800094/d63aabb7-bdbb-4c9b-ae3c-187993925fef)

Reply:
Removed it and reverted to old code.

though I think it would have been better to log only the current experiment logs to mlflow.